docs/platforms/javascript/common/index.mdx

```diff
@@ -37,11 +37,30 @@ If you prefer to follow video instructions, see [How to Install the Sentry Next.
 
 </PlatformSection>
 
-
 <PlatformSection notSupported={["javascript.deno"]}>
 
 ## Install
 
+<PlatformCategorySection notSupported={["server"]}>
```

Comment:
This is all a bit complicated, but it makes sense:

1. We want to show this only for browser-only SDKs, not for any SDK that _also_ has a browser aspect
2. We also don't show it for cordova and plain JavaScript